2016-04-08.md

```diff
@@ -0,0 +1,37 @@
+We are thrilled to announce that Tessel 2 has shipped to all domestic and international pre-orders.
+
+To our knowledge, this is the first time a distributed, unpaid team of open source contributors has shipped a physical product. This is a major milestone in the open source hardware movement.
```

Comment:
It would be great if it were provably true but I don't I think this isn't a very defensible claim in light of Arduino and RasPi's history. I suggest we strike it.